src/match_taxa_gbif_backbone.Rmd

```diff
@@ -0,0 +1,471 @@
+                   )
+)
+```
+
+# Match taxa to GBIF Taxonomy Backbone
+
+## Match by `acceptedname` + `scientificnameauthorship`
+
+We try to match by using `acceptedname` + `scientificnameauthorship`.
+First, we prepare `taxa` for matching. In this step:
+
+1. We create a new column, `name_for_gbif`, where we paste `acceptedname` and `scientificnameauthorship`, if present.
+2. Recode the taxon ranks from `taxonranken` in a new column, `rank_for_gbif`, containing the correct GBIF terms
+
+```{r prepare_taxa_for_matching_backbone}
+taxa <-
+  taxa %>%
+  mutate(name_for_gbif = if_else(
+    !is.na(scientificnameauthorship),
+    paste(acceptedname, scientificnameauthorship),
+    acceptedname
+  )) %>%
+  mutate(rank_for_gbif = recode(taxonranken,
+    "forma" = "form",
+    "subforma" = "subform",
+    "informal group" = NA_character_
+  ))
+```
+
+We match the names in `name_for_gbif` with GBIF Backbone. We also use the columns containing kingdom information, `bim_kingdom`, and rank, `rank_for_gbif`, and we require a strict match, `strict = TRUE` to avoid fuzzy match with higher classification. **Note: This step takes long**:
+
+```{r get_name_gbif}
+taxa_gbif <-
+  taxa %>%
+  gbif_species_name_match(name = "name_for_gbif",
+                          kingdom = "bim_kingdom", 
+                          rank = "rank_for_gbif",
+                          strict = TRUE)
+```
+
+Some columns containing GBIF data have been added. Preview:
+
+```{r overview}
+taxa_gbif %>%
+  head()
+```
+
+The column `matchType` informs us whether the match has been successful, `matchType` = `EXACT`, doubtful, `matchType` = `FUZZY`, or failed, `matchType` = `NONE`. The column  `confidence` is also informative. Number of taxa for each value of `matchType`:
+
+```{r overview_n_match}
+taxa_gbif %>%
+  group_by(matchType) %>%
+  count()
+```
+
+We create a new data.frame, `taxa_no_exact_match`, for taxa without exact match:
+
+```{r taxa_no_exact_match}
+taxa_no_exact_match <-
+  taxa_gbif %>%
+  filter(matchType != "EXACT") %>%
+  arrange(id)
+```
+
+Preview (only relevant columns shown):
+
+```{r not_matched_taxa}
+taxa_no_exact_match %>%
+  select(-c(speciesbeid, colid, vernacularnamefr, 
+            vernacularnamenl, parentid, distributionstatuscode, 
+            media, speciesbeparentid, bruenvi_created, bruenvi_author,
+            bruenvi_modified, kingdom_id)) %>%
+  head()
+```
+
+### Match by `acceptedname` only
+
+Some of the taxa are not matched due to deviating author spellings, e.g.
+
+```{r example_no_match_due_to_author_spelling_deviation}
+taxa_no_exact_match %>%
+  filter(name_for_gbif == "Neosartorya fischeri var. glabra (Fennell & Raper) Malloch & Cain")
+```
+
+while in GBIF Backbone:
+
+```{r}
+rgbif::name_backbone(name = "`Neosartorya fischeri var. glabra Fennell & Raper, 1973`") %>% as_tibble()
+```
+
+Matching without authors can improve matching:
+
+```{r match_acceptedname_only}
+taxa_gbif_acceptedname <-
```

Comment:
I would use another name for this tibble, in line with the flags you give to matchTypes: 
`taxa`
`taxa_without_author` (instead of `taxa_gbif_acceptedname`)
`taxa_without_kingdom` (instead of `taxa_gbif_without_kingdom`)
`taxa_without_author_without_kingdom` (instead of `taxa_gbif_acceptedname_without_kingdom`)